internal/outofband/action_handlers.go

```diff
@@ -515,6 +518,15 @@ func (h *actionHandler) pollFirmwareTaskStatus(a sw.StateSwitch, c sw.Transition
 	// a new collection should be attempted.
 	var inventory bool
 
+	// verifyAttempts is the number of times this poller will spend
+	// attempting to verify the installed firmware equals the expected.
+	//
+	// Multiple attempts to verify is required to allow the BMC time to have its information updated,
+	// the Supermicro BMCs on X12SPO-NTFs, complete the update process, but take
+	// a while to update the installed firmware information returned over redfish.
+	var verifyAttempts int
+	var maxVerifyAttempts = 15
```

Comment:
why 15? Just curious.

Reply:
Thanks for raising that, push a new commit with updated values and clearer comments.